wondrous-app/components/Comment/styles.tsx

```diff
@@ -38,6 +38,7 @@ export const CommentItemContainer = styled.div`
 export const CommentProfilePicture = (props) => (
   <SafeImage
     src={props?.src}
+    useNextImage={false}
```

Comment:
small one, can we set useNextImage to false as default prop instead of passing it every time? 

looks awesome!